superset-frontend/src/SqlLab/components/AceEditorWrapper/index.tsx

```diff
@@ -143,6 +151,16 @@ const AceEditorWrapper = ({
 
       currentSelectionCache.current = selectedText;
     });
+    editor.selection.on('changeCursor', () => {
+      const cursor = editor.getCursorPosition();
+      onCursorPositionChange(cursor);
+    });
+
+    const { row, column } = cursorPosition;
+    editor.moveCursorToPosition({ row, column });
+    editor.focus();
+    editor.renderer.updateFontSize();
```

Comment:
This operation(updateFontSize) seems redundant.

Reply:
You're right. I already modify the code to remove that line of code.